.github/workflows/distribute-grain.yml

```diff
@@ -0,0 +1,61 @@
+name: Distribute Grain
+on:
+  # A new Cred interval every Sunday. We'll distribute Grain 5 minutes
+  # after the new interval
+  schedule:
+    - cron: 5 0 * * 0 # 00:05 UTC, 16:05 PST
+  push:
+    branches:
+      - "graintest-*" # won't actually update ledger or deploy frontend
+
+jobs:
+  distribute-grain:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+
+      - name: "[debug] Check GitHub event name"
+        # This is used to determine whether we actually deploy to GitHub
+        # Pages, so we print it upfront to aid with debugging.
+        run: echo ${{ github.event_name }}
+
+      - name: Install Packages 🔧
+        run: |
+          yarn --frozen-lockfile
+
+      - name: Load Data into Graph 🔌
+        run: |
+          yarn load
+          yarn graph
+        env:
+          SOURCECRED_GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+          SOURCECRED_DISCORD_TOKEN: ${{ secrets.SOURCECRED_DISCORD_TOKEN }}
+
+      - name: Compute Cred 🧮
+        run: |
+          yarn score
+
+      - name: Distribute Grain 💸
+        run: |
+          yarn grain
+
+      - name: Set environment variables
+        run: |
+          echo ::set-env name=PULL_REQUEST_TITLE::"Scheduled grain distribution for week ending $(date +"%B %dth, %Y")"
+          echo ::set-env name=PULL_REQUEST_BODY::"This PR was auto-generated on $(date +%d-%m-%Y) \
+            to add the latest grain distribution to our instance."
+
+      - name: Create commit and PR for ledger changes
+        id: pr
+        uses: peter-evans/create-pull-request@v3
+        with:
+          branch: generated-ledger
+          branch-suffix: timestamp
+          committer: credbot <credbot@users.noreply.github.com>
+          # author appears to be overridden when the default github action
+          # token is used for checkout
+          author: credbot <credbot@users.noreply.github.com>
+          commit-message: update calculated ledger
+          title: ${{ env.PULL_REQUEST_TITLE }}
+          body: ${{ env.PULL_REQUEST_BODY }}
+          reviewers: decentralion
```

Comment:
@decentralion in order to request a review from a team we'll need to use a repo-scoped PAT (a token that "repo" access). This might be an API bug, since repo access seems quite broad for such an action: https://github.com/peter-evans/create-pull-request/issues/155#issuecomment-611904487

Should we configure this for this PR or leave you hard-coded as the reviewer for now

Reply:
Let's leave me as the hard coded reviewer. When we backport to template instance, we can leave it commented out with instructions on how to customize with specific reviewers.